awsagentprovider/src/main/java/software/amazon/opentelemetry/javaagent/providers/AwsAppSignalsCustomizerProvider.java

```diff
@@ -63,29 +66,24 @@ public void customize(AutoConfigurationCustomizer autoConfiguration) {
     autoConfiguration.addSpanExporterCustomizer(this::customizeSpanExporter);
   }
 
-  private boolean isSmpEnabled(ConfigProperties configProps) {
+  private boolean isAppSignalsEnabled(ConfigProperties configProps) {
     return configProps.getBoolean(
         "otel.aws.app.signals.enabled", configProps.getBoolean("otel.smp.enabled", false));
   }
 
   private Sampler customizeSampler(Sampler sampler, ConfigProperties configProps) {
-    if (isSmpEnabled(configProps)) {
+    if (isAppSignalsEnabled(configProps)) {
       return AlwaysRecordSampler.create(sampler);
     }
     return sampler;
   }
 
   private SdkTracerProviderBuilder customizeTracerProviderBuilder(
       SdkTracerProviderBuilder tracerProviderBuilder, ConfigProperties configProps) {
-    if (isSmpEnabled(configProps)) {
+    if (isAppSignalsEnabled(configProps)) {
       logger.info("Span Metrics Processor enabled");
-      String smpEndpoint =
-          configProps.getString(
-              "otel.aws.app.signals.exporter.endpoint",
-              configProps.getString("otel.aws.smp.exporter.endpoint", "http://localhost:4315"));
       Duration exportInterval =
           configProps.getDuration("otel.metric.export.interval", DEFAULT_METRIC_EXPORT_INTERVAL);
-      logger.log(Level.FINE, String.format("Span Metrics endpoint: %s", smpEndpoint));
       logger.log(Level.FINE, String.format("Span Metrics export interval: %s", exportInterval));
```

Comment:
Change the log statement to "CW Application Signals metrics export interval:..."?